pkg/git/repository/gitlab/service.go

```diff
@@ -0,0 +1,88 @@
+package gitlab
+
+import (
+	"github.com/redhat-developer/git-service/pkg/apis/devconsole/v1alpha1"
+	"github.com/redhat-developer/git-service/pkg/git"
+	"github.com/redhat-developer/git-service/pkg/git/repository"
+	gogl "github.com/xanzy/go-gitlab"
+	gittransport "gopkg.in/src-d/go-git.v4/plumbing/transport"
+)
+
+const (
+	gitlabHost   = "gitlab.com"
+	gitlabFlavor = "gitlab"
+)
+
+type RepositoryService struct {
+	client *gogl.Client
+	repo   repository.StructuredIdentifier
+}
+
+func NewRepoServiceIfMatches() repository.ServiceCreator {
```

Comment:
Would be nice if we could add some comments - especially for public functions

Reply:
good point :-). is it better? f016f77

Reply:
awesome :)